glacier/src/test/importer.spec.ts

```diff
@@ -267,4 +269,37 @@ describe("glacier as a model", () => {
         await baseline("5-Product Weight", await exporter.export()); // NOT A BUG - uses the same baseline as the fifth baseline
         await adapter.remove();
     });
+    it("should create export state to disc", async() => {
+        let model = glacier.createModel();
+        const adapter = glacier.createSqlFileDataSource(model, "../data/CycleChain.sqlite");
+        const addFields = [{name: "DaysToManufacture", table: "Product", dataSource: adapter.id}, {name: "ListPrice", table: "Product", dataSource: adapter.id}, {name: "Weight", table: "Product", dataSource: adapter.id}];
+        const removeFields = [{name: "ListPrice", table: "Product", dataSource: adapter.id}];
+        dispatchSequence(model,
+            glacier.createAddFieldsAction(addFields),
+            glacier.createRemoveFieldsAction(removeFields),
+            glacier.createUpdateMarkTypeAction("bar"),
+            glacier.createUpdateSizeAction(255, 264),
+            glacier.createUpdateEncodingAction({
+                x: {field: "DaysToManufacture", type: "ordinal", scale: {bandSize: 20}},
+                y: {field: "Weight", type: "quantitative"},
+        })
+        );
+
+        // const exporter = glacier.createSvgExporter(model, adapter.id);
+        const lib = fs.readFileSync(resolve(__dirname, "../../dist/local/glacier.js"));
+        const exportedState = glacier.createStateExporter(model, adapter.id, lib);
+        await adapter.updateCache();
+        const zip = await exportedState.export();
+        zip.generateNodeStream({type: "nodebuffer", streamFiles: true})
+        .pipe(fs.createWriteStream("../data/out.zip"));
+
+        jszip.loadAsync(fs.readFileSync(resolve(__dirname, "../../../data/out.zip"))).then(function(zip: JSZip){
```

Comment:
Use `async` functions and `await` to reduce the nesting inside this call, please. (Everywhere you do a `.then`, await the thing you're `then`'ing and assign it to a value)
For example:
```ts
const zip = await jszip.loadAsync(fs.readFileSync(resolve(__dirname, "../../../data/out.zip")));
```